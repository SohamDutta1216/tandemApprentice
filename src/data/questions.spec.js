const questions = require('./questions.json')

test('our questions have not changed since initial data was loaded', () => {
  expect(questions).toMatchSnapshot()
})

test('our questions data has a length greater than 0', () => {
  let currentLength = questions.length
  expect(currentLength).toBeGreaterThan(0)
})

test('our questions are defined', () => {
  expect(questions.map(quest => quest.question)).toBeDefined()
})

test('our incorrect answers are defined', () => {
  expect(questions.map(quest => quest.incorrect)).toBeDefined()
})

test('our correct answers are defined', () => {
  expect(questions.map(quest => quest.correct)).toBeDefined()
})
