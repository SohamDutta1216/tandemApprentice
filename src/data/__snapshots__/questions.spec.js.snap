// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`our questions data is defined 1`] = `
Array [
  Object {
    "correct": "Devmynd",
    "incorrect": Array [
      "Tandem",
      "Burger Shack",
      "Extraordinary Humans",
    ],
    "question": "What was Tandem previous name?",
  },
  Object {
    "correct": "Et tu, Brute?",
    "incorrect": Array [
      "Iacta alea est!",
      "Vidi, vini, vici",
      "Aegri somnia vana",
    ],
    "question": "In Shakespeare's play Julius Caesar, Caesar's last words were...",
  },
  Object {
    "correct": "Ambush",
    "incorrect": Array [
      "Chowder",
      "Pride",
      "Destruction",
    ],
    "question": "A group of tigers are referred to as:",
  },
  Object {
    "correct": "31 mph",
    "incorrect": Array [
      "42 mph",
      "13 mph",
      "9 mph",
    ],
    "question": "What is the top speed an average cat can travel?",
  },
  Object {
    "correct": "5",
    "incorrect": Array [
      "3",
      "9",
      "7",
    ],
    "question": "A cat can jump to _____ times its own height:",
  },
  Object {
    "correct": "Q",
    "incorrect": Array [
      "M",
      "Z",
      "X",
    ],
    "question": "What is the only letter that doesn't appear in a US state name?",
  },
  Object {
    "correct": "Beefalo",
    "incorrect": Array [
      "Cowson",
      "Bicow",
      "Mooson",
    ],
    "question": "What is the name for a cow-bison hybrid?",
  },
  Object {
    "correct": "Lake Superior",
    "incorrect": Array [
      "Lake Baikal",
      "Lake Michigan",
      "Lake Victoria",
    ],
    "question": "What is the largest freshwater lake in the world?",
  },
  Object {
    "correct": "World Wide Web",
    "incorrect": Array [
      "Wild Wild West",
      "War World Web",
    ],
    "question": "In a website address bar, what does WWW stand for?",
  },
  Object {
    "correct": "22",
    "incorrect": Array [
      "20",
      "55",
      "77",
    ],
    "question": "In a game of bingo, what number is represented by the name two little ducks?",
  },
  Object {
    "correct": "Pandora",
    "incorrect": Array [
      "Lilith",
      "Eve",
      "Hera",
    ],
    "question": "According to Greek mythology, who was the first woman on Earth?",
  },
  Object {
    "correct": "Paris",
    "incorrect": Array [
      "London",
      "Belgium",
      "Munich",
    ],
    "question": "In which European city would you find Orly airport?",
  },
  Object {
    "correct": "The Moon",
    "incorrect": Array [
      "California",
      "Siberia",
      "China",
    ],
    "question": "Where would you find the Sea of Tranquility?",
  },
  Object {
    "correct": "Vermeer",
    "incorrect": Array [
      "Van Gogh",
      "Picasso",
      "Da Vinci",
    ],
    "question": "Which artist painted 'Girl with a Pearl Earrin'?",
  },
  Object {
    "correct": "Octothorpe",
    "incorrect": Array [
      "Number sign",
      "Hash Sign",
      "Pound",
    ],
    "question": "What is the official name for the 'hashtag' symbol?",
  },
  Object {
    "correct": "England",
    "incorrect": Array [
      "Japan",
      "Ethiopia",
      "Canada",
    ],
    "question": "Not American at all, where is apple pie from?",
  },
  Object {
    "correct": "Unicorn",
    "incorrect": Array [
      "Bear",
      "Rabbit",
      "Seal",
    ],
    "question": "What is the national animal of Scotland?",
  },
  Object {
    "correct": "Detroit",
    "incorrect": Array [
      "Alaska",
      "Russia",
      "Washington",
    ],
    "question": "Where in the world is the only place where Canada is *due south*",
  },
  Object {
    "correct": "700",
    "incorrect": Array [
      "500",
      "200",
      "1000",
    ],
    "question": "Approximately how many grapes go into a bottle of wine?",
  },
  Object {
    "correct": "$0.05",
    "incorrect": Array [
      "$0.25",
      "$1",
      "$5",
    ],
    "question": "How much does a US One Dollar Bill cost to make?",
  },
  Object {
    "correct": "Perform transactions in Latin",
    "incorrect": Array [
      "Receive withdrawls in rosary beads",
      "Vote for the Pope",
      "Purchase indulgences",
    ],
    "question": "The Vatican bank has the only ATM in the world that allows users to do what?",
  },
]
`;

exports[`our questions data is valid 1`] = `
Array [
  Object {
    "correct": "Devmynd",
    "incorrect": Array [
      "Tandem",
      "Burger Shack",
      "Extraordinary Humans",
    ],
    "question": "What was Tandem previous name?",
  },
  Object {
    "correct": "Et tu, Brute?",
    "incorrect": Array [
      "Iacta alea est!",
      "Vidi, vini, vici",
      "Aegri somnia vana",
    ],
    "question": "In Shakespeare's play Julius Caesar, Caesar's last words were...",
  },
  Object {
    "correct": "Ambush",
    "incorrect": Array [
      "Chowder",
      "Pride",
      "Destruction",
    ],
    "question": "A group of tigers are referred to as:",
  },
  Object {
    "correct": "31 mph",
    "incorrect": Array [
      "42 mph",
      "13 mph",
      "9 mph",
    ],
    "question": "What is the top speed an average cat can travel?",
  },
  Object {
    "correct": "5",
    "incorrect": Array [
      "3",
      "9",
      "7",
    ],
    "question": "A cat can jump to _____ times its own height:",
  },
  Object {
    "correct": "Q",
    "incorrect": Array [
      "M",
      "Z",
      "X",
    ],
    "question": "What is the only letter that doesn't appear in a US state name?",
  },
  Object {
    "correct": "Beefalo",
    "incorrect": Array [
      "Cowson",
      "Bicow",
      "Mooson",
    ],
    "question": "What is the name for a cow-bison hybrid?",
  },
  Object {
    "correct": "Lake Superior",
    "incorrect": Array [
      "Lake Baikal",
      "Lake Michigan",
      "Lake Victoria",
    ],
    "question": "What is the largest freshwater lake in the world?",
  },
  Object {
    "correct": "World Wide Web",
    "incorrect": Array [
      "Wild Wild West",
      "War World Web",
    ],
    "question": "In a website address bar, what does WWW stand for?",
  },
  Object {
    "correct": "22",
    "incorrect": Array [
      "20",
      "55",
      "77",
    ],
    "question": "In a game of bingo, what number is represented by the name two little ducks?",
  },
  Object {
    "correct": "Pandora",
    "incorrect": Array [
      "Lilith",
      "Eve",
      "Hera",
    ],
    "question": "According to Greek mythology, who was the first woman on Earth?",
  },
  Object {
    "correct": "Paris",
    "incorrect": Array [
      "London",
      "Belgium",
      "Munich",
    ],
    "question": "In which European city would you find Orly airport?",
  },
  Object {
    "correct": "The Moon",
    "incorrect": Array [
      "California",
      "Siberia",
      "China",
    ],
    "question": "Where would you find the Sea of Tranquility?",
  },
  Object {
    "correct": "Vermeer",
    "incorrect": Array [
      "Van Gogh",
      "Picasso",
      "Da Vinci",
    ],
    "question": "Which artist painted 'Girl with a Pearl Earrin'?",
  },
  Object {
    "correct": "Octothorpe",
    "incorrect": Array [
      "Number sign",
      "Hash Sign",
      "Pound",
    ],
    "question": "What is the official name for the 'hashtag' symbol?",
  },
  Object {
    "correct": "England",
    "incorrect": Array [
      "Japan",
      "Ethiopia",
      "Canada",
    ],
    "question": "Not American at all, where is apple pie from?",
  },
  Object {
    "correct": "Unicorn",
    "incorrect": Array [
      "Bear",
      "Rabbit",
      "Seal",
    ],
    "question": "What is the national animal of Scotland?",
  },
  Object {
    "correct": "Detroit",
    "incorrect": Array [
      "Alaska",
      "Russia",
      "Washington",
    ],
    "question": "Where in the world is the only place where Canada is *due south*",
  },
  Object {
    "correct": "700",
    "incorrect": Array [
      "500",
      "200",
      "1000",
    ],
    "question": "Approximately how many grapes go into a bottle of wine?",
  },
  Object {
    "correct": "$0.05",
    "incorrect": Array [
      "$0.25",
      "$1",
      "$5",
    ],
    "question": "How much does a US One Dollar Bill cost to make?",
  },
  Object {
    "correct": "Perform transactions in Latin",
    "incorrect": Array [
      "Receive withdrawls in rosary beads",
      "Vote for the Pope",
      "Purchase indulgences",
    ],
    "question": "The Vatican bank has the only ATM in the world that allows users to do what?",
  },
]
`;

exports[`our questions have not changed since initial data was loaded 1`] = `
Array [
  Object {
    "correct": "Devmynd",
    "incorrect": Array [
      "Tandem",
      "Burger Shack",
      "Extraordinary Humans",
    ],
    "question": "What was Tandem previous name?",
  },
  Object {
    "correct": "Et tu, Brute?",
    "incorrect": Array [
      "Iacta alea est!",
      "Vidi, vini, vici",
      "Aegri somnia vana",
    ],
    "question": "In Shakespeare's play Julius Caesar, Caesar's last words were...",
  },
  Object {
    "correct": "Ambush",
    "incorrect": Array [
      "Chowder",
      "Pride",
      "Destruction",
    ],
    "question": "A group of tigers are referred to as:",
  },
  Object {
    "correct": "31 mph",
    "incorrect": Array [
      "42 mph",
      "13 mph",
      "9 mph",
    ],
    "question": "What is the top speed an average cat can travel?",
  },
  Object {
    "correct": "5",
    "incorrect": Array [
      "3",
      "9",
      "7",
    ],
    "question": "A cat can jump to _____ times its own height:",
  },
  Object {
    "correct": "Q",
    "incorrect": Array [
      "M",
      "Z",
      "X",
    ],
    "question": "What is the only letter that doesn't appear in a US state name?",
  },
  Object {
    "correct": "Beefalo",
    "incorrect": Array [
      "Cowson",
      "Bicow",
      "Mooson",
    ],
    "question": "What is the name for a cow-bison hybrid?",
  },
  Object {
    "correct": "Lake Superior",
    "incorrect": Array [
      "Lake Baikal",
      "Lake Michigan",
      "Lake Victoria",
    ],
    "question": "What is the largest freshwater lake in the world?",
  },
  Object {
    "correct": "World Wide Web",
    "incorrect": Array [
      "Wild Wild West",
      "War World Web",
    ],
    "question": "In a website address bar, what does WWW stand for?",
  },
  Object {
    "correct": "22",
    "incorrect": Array [
      "20",
      "55",
      "77",
    ],
    "question": "In a game of bingo, what number is represented by the name two little ducks?",
  },
  Object {
    "correct": "Pandora",
    "incorrect": Array [
      "Lilith",
      "Eve",
      "Hera",
    ],
    "question": "According to Greek mythology, who was the first woman on Earth?",
  },
  Object {
    "correct": "Paris",
    "incorrect": Array [
      "London",
      "Belgium",
      "Munich",
    ],
    "question": "In which European city would you find Orly airport?",
  },
  Object {
    "correct": "The Moon",
    "incorrect": Array [
      "California",
      "Siberia",
      "China",
    ],
    "question": "Where would you find the Sea of Tranquility?",
  },
  Object {
    "correct": "Vermeer",
    "incorrect": Array [
      "Van Gogh",
      "Picasso",
      "Da Vinci",
    ],
    "question": "Which artist painted 'Girl with a Pearl Earrin'?",
  },
  Object {
    "correct": "Octothorpe",
    "incorrect": Array [
      "Number sign",
      "Hash Sign",
      "Pound",
    ],
    "question": "What is the official name for the 'hashtag' symbol?",
  },
  Object {
    "correct": "England",
    "incorrect": Array [
      "Japan",
      "Ethiopia",
      "Canada",
    ],
    "question": "Not American at all, where is apple pie from?",
  },
  Object {
    "correct": "Unicorn",
    "incorrect": Array [
      "Bear",
      "Rabbit",
      "Seal",
    ],
    "question": "What is the national animal of Scotland?",
  },
  Object {
    "correct": "Detroit",
    "incorrect": Array [
      "Alaska",
      "Russia",
      "Washington",
    ],
    "question": "Where in the world is the only place where Canada is *due south*",
  },
  Object {
    "correct": "700",
    "incorrect": Array [
      "500",
      "200",
      "1000",
    ],
    "question": "Approximately how many grapes go into a bottle of wine?",
  },
  Object {
    "correct": "$0.05",
    "incorrect": Array [
      "$0.25",
      "$1",
      "$5",
    ],
    "question": "How much does a US One Dollar Bill cost to make?",
  },
  Object {
    "correct": "Perform transactions in Latin",
    "incorrect": Array [
      "Receive withdrawls in rosary beads",
      "Vote for the Pope",
      "Purchase indulgences",
    ],
    "question": "The Vatican bank has the only ATM in the world that allows users to do what?",
  },
]
`;
